src/Sylius/Behat/Exception/NotificationExpectationMismatchException.php

```diff
@@ -11,33 +11,35 @@
 
 namespace Sylius\Behat\Exception;
 
+use Sylius\Behat\Service\NotificationType;
+
 /**
  * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
  */
 final class NotificationExpectationMismatchException extends \RuntimeException
 {
     /**
-     * @param string $missingNotificationType
-     * @param string $missingNotification
-     * @param string $foundedNotificationType
-     * @param string $foundedNotification
+     * @param NotificationType $missingType
+     * @param string $missingMessage
+     * @param NotificationType $foundedType
+     * @param string $foundedMessage
      * @param int $code
      * @param \Exception|null $previous
      */
     public function __construct(
-        $missingNotificationType,
-        $missingNotification,
-        $foundedNotificationType,
-        $foundedNotification,
-        $code = 0, 
+        NotificationType $missingType,
+        $missingMessage,
+        NotificationType $foundedType,
```

Comment:
I would rename this parameters to:
* `$expectedType`, `$expectedMessage`
* `$actualType`, `$actualMessage`